sematic/api/endpoints/auth.py

```diff
@@ -43,7 +43,7 @@ def authenticate_endpoint() -> flask.Response:
             # ServerSettingsVar.GITHUB_OAUTH_CLIENT_ID,
         ):
             try:
-                providers[var.value] = get_server_settings(var)
+                providers[var.value] = get_server_setting(var)
             except MissingServerSettingsError:
```

Comment:
`MissingServerSettingsError` is never raised anymore. `MissingUserSettingsError` is now only raised in `api_client.py`. The base class `MissingSettingsError` is effectively raised instead in `settings.py`.

Please just drop `MissingServerSettingsError` and `MissingUserSettingsError` and use only `MissingSettingsError` everywhere, and manually test missing settings cases.

Reply:
Done.